### ISSUE TITLE
when reply in the plugin reply box. messages do not go out with SMIME encryption set

### ISSUE THREAD

**Issue by Maskedman** [protz/GMail-Conversation-View#996]
In the reply box, if the original sender sends me a message encrypted using SMIME, my reply should automatically use my SMIME certificates to encrypt messages back to sender.

If I open the original message in its own window, Thunderbird preserves the encryption through to the reply window.

**Comment by protz** [comment]
Correct, Conversations does not have S/MIME support. If you have the skills, I'd be happy to see you write a Conversations plugin for that. Otherwise, given the limited time that I currently have, it's unlikely to happen...
